### PR TITLE
[Refactoring] Rename Xiaomi to Lumi in lookup

### PR DESCRIPTION
**Pull request by mrskycriper**
Renamed Xiaomi to Lumi in manufacturer lookup in continuation of this [PR](https://github.com/Koenkk/zigbee-herdsman-converters/pull/6969)
 
Also added missing manifacturer lookup for Perenio (`4456: 'Perenio'`).

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -898,9 +898,9 @@
             "_TZE200_cpmgn2cf"
         ],
         "imageType": 5634,
-        "sha512": "1113486a67403d922fc4a462b7477e0a0986fa8862a89b74ab8c5eac7b4ecb31c30f1da9ae1bbc1d6d50ae1fed66d38c37a1834ab71fd384c0555002783aba13",
-        "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Tuya/1615190575-si32_zg_uart_connect_sleep_ZS5_ty_OTA_1.1.7.bin",
-        "path": "images/Tuya/1615190575-si32_zg_uart_connect_sleep_ZS5_ty_OTA_1.1.7.bin"
+        "sha512": "41ebf9932d11708b81144232b2c3fccc7a58d76116be63785b978b91afb1180e32d8ba4f4c8f2f0960637f1662ec9ac28aa42b74e5ab57574000889671d2b8b9",
```
Comment (Koenkk) [inline index.json:901]: I see some unexpected changes in this PR, why was this changed?

File: index.json [hunk 2]
```diff
@@ -2124,8 +2125,8 @@
         "imageType": 4361,
         "sha512": "dd7ee3b0161bcc819308004e550877adfec1adde7f140bd42835aeba386bb9105b9002f7e72f6e2563e8c1135a8f0736d629a3107094bab74c9c9ce34826b99b",
         "modelId": "lumi.light.acn004",
-        "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Xiaomi/20230908121913_OTA_lumi.light.acn004_0.0.0_0030_20230908_D28C7F.ota",
-        "path": "images/Xiaomi/20230908121913_OTA_lumi.light.acn004_0.0.0_0030_20230908_D28C7F.ota"
+        "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Lumi/20230908121913_OTA_lumi.light.acn004_0.0.0_0030_20230908_D28C7F.ota",
```
Comment (Koenkk) [inline index.json:2128]: Can you remove all the old OTA files.

File: index.json (hunk 1 above)
Comment (mrskycriper) [reply index.json:901]: I rerun `updateall.js` as instructed, probably because of this

File: index.json (hunk 2 above)
Comment (mrskycriper) [reply index.json:2128]: Removed Xiaomi folder

File: index.json (hunk 1 above)
Comment (mrskycriper) [reply index.json:901]: It looks like add.js does not check `manufacturerName` at all when deciding if it is a duplucate. 

https://github.com/Koenkk/zigbee-OTA/blob/930f4cef3575ef876d318553c7355ce6065f70e9/scripts/add.js#L135

Comment (mrskycriper) [reply index.json]: Reverted this change manually for now.

Comment (Koenkk) [reply index.json]: The `updateall.js` is indeed not correct anymore and should be fixed. In this PR can you just have the following changes: 
- Rename `images/Xiaomi` to `images/Lumi`
- Update the paths in `index.json` to reflect this.

Comment (mrskycriper) [reply index.json]: So, all remove other updates?

**Comment by Koenkk** [comment]
Thanks!
